tests/tasks/create_test_interfaces.yml

```
@@ -0,0 +1,42 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- name: Install dnsmasq
+  package:
+    name: dnsmasq
+    state: present
+
+
+- name: Create test interfaces
+  shell: |
+    # Manage all veth devices starting with test*
+    echo 'ENV{ID_NET_DRIVER}=="veth", ENV{INTERFACE}=="test*", ENV{NM_UNMANAGED}="0"' >/etc/udev/rules.d/88-veth.rules
+    udevadm control --reload-rules
+    udevadm control --reload-rules
+
+    # Create a namespace for devices
+    ip netns add testsetup
+
+    # Setup simulated device test1 and test2 with IPv6 auto support
+    ip link add test1 type veth peer name test1p
```

```suggestion
    ip link add {{dhcp_interface1}} type veth peer name {{dhcp_interface1}}p
```
Then you need to use `dhcp_interface1` instead of `slave1_device` in the vars and would not need to hardcode the interface name.

---

ok, using this as you suggested